core/core-funcs.el

```diff
@@ -128,6 +128,11 @@ Supported properties:
     (show-all))
    (t nil))
 
+  ;; Make ~SPC ,~ work, reference:
+  ;; http://stackoverflow.com/questions/24169333/how-can-i-emphasize-or-verbatim-quote-a-comma-in-org-mode
+  (setcar (nthcdr 2 org-emphasis-regexp-components) " \t\n")
+  (org-set-emph-re 'org-emphasis-regexp-components org-emphasis-regexp-components)
```

Comment:
The commit the title was referring to should be here: https://github.com/syl20bnr/spacemacs/pull/1968